backend/application/import_observations/parsers/prometheus_trivy/parser.py

```diff
@@ -0,0 +1,167 @@
+            description = self.get_description(
+                component_version, vulnerability_id, cvss_v3_base_score, vuln_title, image, component
+            )
+
+            state = ""
+            reference_url = ""
+            cwes = ""
+
+            observation = Observation(
+                title=vulnerability_id,
+                description=description,
+                parser_severity=self.get_severity(severity),
+                parser_status=self.get_status(state),
+                vulnerability_id=vulnerability_id,
+                origin_component_name=component_name,
+                origin_component_version=component_version,
+                cvss3_score=cvss_v3_base_score,
+                cwe=self.get_cwe(cwes),
+                scanner=scanner,
+            )
+
+            evidence = []
+            evidence.append("Vulnerability")
+            evidence.append(json.dumps(finding))
+            observation.unsaved_evidences.append(evidence)
+
+            if reference_url:
+                observation.unsaved_references = [reference_url]
+
+            observations.append(observation)
+
+        return observations
+    
+    def get_description(  # pylint: disable=too-many-branches
+        self,
+        component_version: str,
+        vulnerability_id: str,
+        cvss_v3_base_score: str,
+        vuln_title: str,
+        image: str,
+        component: str,
+    ) -> str:
+        description = ""
+
+        description += f"**Component**: {component}\n\n"
+        description += f"**Rule full description**: {vuln_title}\n\n"
+        description += f"**Security-Severity**: {cvss_v3_base_score}\n\n"
+        description += f"**CVE**: [{vulnerability_id}](https://nvd.nist.gov/vuln/detail/{vulnerability_id})\n\n"
+        description += f"**Image**: {image}:{component_version}\n\n"
+
+        return description
+
+    def get_status(self, state: str) -> str:
+        if not state:
+            return ""
+
+        return STATUS_MAPPING.get(state, "")
+
+    def get_severity(self, severity: str) -> str:
+        if (
+            severity.capitalize(),
+            severity.capitalize(),
+        ) in Severity.SEVERITY_CHOICES:
+            return severity.capitalize()
+
+        return Severity.SEVERITY_UNKOWN
+
+    def get_cwe(self, cwes: list[dict]) -> int | None:
+        if cwes:
+            return cwes[0].get("cweId")
+
+        return None
+
+    def get_about(self) -> tuple[str, Optional[str]]:
+        if not self.api_configuration:
+            return "Prometheus-Trivy", None
+
+        prometheus_trivy_base_url = self.api_configuration.base_url
+        if not prometheus_trivy_base_url.endswith("/"):
+            prometheus_trivy_base_url += "/"
+
+        try:
+            response = requests.get(prometheus_trivy_base_url, timeout=60)
+            response.raise_for_status()
+        except Exception:
+            return "Prometheus-Trivy", None
+
+        application = "Prometheus-Trivy"
```

Comment:
Why calling Prometheus first and then setting `application` and `version` manually?

Reply:
Just to get it running. Will be changed later